tests/e2e/matriculationManagement.spec.ts

```diff
@@ -26,29 +28,39 @@ describe("Account creation, edition and deletion", function () {
             (a as Admin).username += random;
             admin = a as Admin;
         });
-        cy.fixture("adminAuthUser.json").then((a) => {
-            (a as Account).username += random;
-            adminAuthUser = a as Account;
-        });
-
-        cy.fixture("student.json").then((s) => {
-            (s as Student).username += random;
-            student = s as Student;
-            var today = new Date();
-            var monthPadded = ("00" + (today.getMonth() + 1)).substr(-2);
-            var dayPadded = ("00" + today.getDate()).substr(-2);
-            var random2 = Math.floor(Math.random() * 999).toString();
-            var randomPadded = ("000" + random2).substr(-3);
-            student.matriculationId = monthPadded + dayPadded + randomPadded;
-        });
-        cy.fixture("studentAuthUser.json").then((s) => {
-            (s as Account).username += random;
-            studentAuthUser = s as Account;
-        });
+        cy.fixture("logins/admin.json")
+            .then((admin) => {
+                adminAuth = admin;
+            })
+            .then(async () => {
+                await getMachineUserAuth(adminAuth);
+            })
+            .then(() => {
+                cy.fixture("student.json").then((s) => {
+                    (s as Student).username += random;
+                    student = s as Student;
+                    var today = new Date();
+                    var monthPadded = ("00" + (today.getMonth() + 1)).substr(-2);
+                    var dayPadded = ("00" + today.getDate()).substr(-2);
+                    var random2 = Math.floor(Math.random() * 999).toString();
+                    var randomPadded = ("000" + random2).substr(-3);
+                    student.matriculationId = monthPadded + dayPadded + randomPadded;
```

Comment:
we should add a method for this in a helper "getRandomMatriculationId()"

Reply:
done.